.github/workflows/copilot-setup-steps.yml

```
@@ -17,11 +17,39 @@ jobs:
     steps:
       - uses: actions/checkout@v4.2.2
 
-      - name: Install dependencies
-        run: sudo ./eng/common/native/install-dependencies.sh
-      - name: Restore solution
-        run: ./build.sh --restore --excludecibinarylog --warnaserror false /p:BuildAllConfigurations=true /p:DotNetBuildAllRuntimePacks=true
+      - name: Install Dependencies
+        env:
+          LIBMSQUIC_VERSION: '2.4.8'
+        run: |
+          sudo ./eng/common/native/install-dependencies.sh && \
+          sudo apt-get install -qq -y \
+            curl \
+            libbpf1 \
+            libelf1t64 \
+            libnl-3-200 \
+            libnl-route-3-200 \
+            libnuma1 \
+            libxdp1 && \
+          curl -LO "https://packages.microsoft.com/ubuntu/24.04/prod/pool/main/libm/libmsquic/libmsquic_${LIBMSQUIC_VERSION}_amd64.deb" && \
+          sudo dpkg -i libmsquic* && \
+          rm libmsquic*
+
+      - name: Restore Solution
+        env:
+          RESTORE_ARGS: --restore --excludecibinarylog --warnaserror false /p:BuildAllConfigurations=true /p:DotNetBuildAllRuntimePacks=true
+        # 'DefaultSubsets' don't include 'libs.tests', so restoring it explicitly
+        run: |
+          ./build.sh $RESTORE_ARGS && \
+          ./build.sh libs.tests $RESTORE_ARGS
```

```suggestion
      - name: Restore solution
        run: ./build.sh --restore --excludecibinarylog --warnaserror false /p:BuildAllConfigurations=true /p:DotNetBuildAllRuntimePacks=true /p:DotNetBuildTests=true
```

You can use `DotNetBuildTests`.  We should really have a flag for subsets that means "everything" cc @ViktorHofer